apps/web/src/features/calendar/calendar.css

```
@@ -349,6 +349,29 @@
 	font-size: clamp(0.625rem, min(6cqi, 20cqh), 0.75rem);
 }
 
+.calendar-view .fc .calendar-event-everyone-declined-indicator {
+	background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)